debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java

```diff
@@ -70,6 +73,14 @@ else if ("true".equals(legacyDateTime)) {
             logger.warn("'" + JDBC_PROPERTY_LEGACY_DATETIME + "'" + " is set to 'true'. This setting is not recommended and can result in timezone issues.");
         }
 
+        if (jdbcConfig.getString(JDBC_PROPERTY_VERIFY_SERVER_CERT) != null) {
```

Comment:
Is it actually needed to pass them here? They should be propagated automatically to the JDBC driver when prefixed with the "database." property prefix.

Reply:
Excatly, there is an automatic passthrough so this coude should be superluous.